R/expect_lint.R

```diff
@@ -45,8 +45,10 @@ expect_lint <- function(content, checks, ..., file = NULL, language = "en") {
 
   if (is.null(file)) {
     file <- tempfile()
+    con <- base::file(file, encoding = "UTF-8")
```

Comment:
I usually wouldn't write `base::`

Reply:
`file` is also an argument name, so it's needed here for disambiguation. Otherwise, I agree 😄

Reply:
oh I see